package.json

```diff
@@ -17,68 +17,62 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "dependencies": {
-    "chalk": "^4.1.2",
-    "lit-element": "^2.5.1"
+    "chalk": "^5.2.0",
+    "lit": "^2.7.4"
   },
   "peerDependencies": {
     "@alaskaairux/design-tokens": "^3.3.0",
-    "@alaskaairux/icons": "^4.7.0",
-    "@webcomponents/webcomponentsjs": "^2.5.0",
-    "focus-visible": "^5.2.0"
+    "@alaskaairux/icons": "^4.7.0"
   },
   "devDependencies": {
-    "@alaskaairux/auro-icon": "^3.4.0",
-    "@alaskaairux/design-tokens": "^3.12.2",
-    "@alaskaairux/icons": "^4.21.2",
+    "@alaskaairux/auro-icon": "^3.4.4",
```

Comment:
Deprecated. Update to new version. 

https://www.npmjs.com/package/@alaskaairux/auro-icon

Reply:
on it